org.zaproxy.ZAP.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "org.zaproxy.ZAP",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
```

Comment:
Update runtime to 19.08

Reply:
Not possible for now. Openjsdk still doesn't have a 19.08 branch.